client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-how-to-migrate/index.tsx

```diff
@@ -135,6 +137,37 @@ const SiteMigrationHowToMigrate: Step< {
 		);
 	};
 
+	if ( isUsingStepContainerV2 ) {
+		return (
+			<>
+				<DocumentHead title={ translate( 'Let us migrate your site' ) } />
+				<Step.CenteredColumnLayout
```

Comment:
We explicitly narrowed this layout as part of the recent Code Blue V0 work -- see https://github.com/Automattic/wp-calypso/issues/100861 and https://github.com/Automattic/wp-calypso/pull/101519.

I see that we can either aim for a width of 4 or 6. 6 has a max-width of 615px, and the 4 case is _really_ narrow with a max-width of 404px -- see the screenshot below. The current layout has a max-width almost exactly in the middle of that at 502px. @fditrapani, could you weigh in on what would be good for our use case and/or the `CenteredColumnLayout` wrapper?

![Screenshot 2025-03-26 at 11 15 15](https://github.com/user-attachments/assets/30c44965-2cf7-451c-9230-cce157fd83ee)

Reply:
Thanks for the ping @daledupreez. Unfortunately neither of these options looks great. The wide one is too wide and hinders readability and the other one is too short making it looked cramped. Can we go with the wider one and then set a max width (400px) to the text to help with the readability? Like this:

![image](https://github.com/user-attachments/assets/b2a952cf-f554-4ae5-a1fa-a5dd379c2f1c)

Reply:
> Can we go with the wider one and then set a max width (400px) to the text to help with the readability?
> The current layout has a max-width almost exactly in the middle of that at 502px

I'd prefer not to use custom styles for these blocks since we're aiming to reuse the style of these blocks across these screens.

[I added a new layout-5](https://github.com/Automattic/wp-calypso/pull/101738/commits/e58450bd757e5b1678420bea2927db533e0b7ae0) with a middle size between `layout 4` and `layout 6`. Here's how it looks:

![image](https://github.com/user-attachments/assets/f79e610f-c2d3-450a-9c30-1e892e160bd6)


How does that sound to you @fditrapani and @fcoveram?

Reply:
That looks great to me! thanks @paulopmt1